libs/lib_loaders.py

```diff
@@ -234,12 +236,12 @@ def generate_infractionid() -> str:
 def inc_statistics_better(guild: int, inctype: str, kernel_ramfs: lexdpyk.ram_filesystem) -> None:
 
     try:
-        statistics: Dict[str, int] = kernel_ramfs.read_f(f"{guild}/stats")
+        statistics: dict[str, int] = kernel_ramfs.read_f(f"{guild}/stats")
     except FileNotFoundError:
         statistics = kernel_ramfs.create_f(f"{guild}/stats", f_type=cast(Type[Dict[str, int]], dict))
```

Comment:
sort of confused here why `Dict` and not `dict`

Reply:
3.8 doesnt allow 3.9 syntax outside of `: type` and `-> type` annotations, and thats with `from __future__ import annotations`